arch/arm/core/cortex_m/mpu/arm_mpu.c

```diff
@@ -10,9 +10,45 @@
 #include <soc.h>
 #include <arch/arm/cortex_m/cmsis.h>
 #include <arch/arm/cortex_m/mpu/arm_mpu.h>
+#include <arch/arm/cortex_m/mpu/arm_core_mpu.h>
+#include <logging/sys_log.h>
 
 #define ARM_MPU_DEV ((volatile struct arm_mpu *) ARM_MPU_BASE)
 
+/* ARM MPU Enabled state */
+static u8_t arm_mpu_enabled;
+
+/**
+ * The attributes referenced in this function are described at:
+ * https://goo.gl/hMry3r
```

Comment:
I'd recommend not using shortened urls, but the full url is longer than allowed by our coding standards:
http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0553a/Cihegaib.html
sigh.  I guess we're stuck with the shortened url :(